specs/http-client2/index.html

```diff
@@ -181,6 +181,10 @@ <h3>http:put</h3>
           Sends a PUT request with an optional <code>$body</code> to the supplied <code>$uri</code>
           and returns the response as a map. Additional <code>$options</code> can be supplied.
         </p>
+        <p>
+          HTTP PUT is only supported in HTTP 1.1. If <code>"http-version": "1.0"</code> is set in
+          the <code>$options</code> then TODO-ERROR-CODE is raised.
```

Comment:
Looks good. It could possibly be named `error:version`.

Reply:
Sure. I was wondering about Error Codes generally. Should we for now just mark them all `TODO-ERROR-CODE`?
Then near the end of the spec writing process, we could collect them up and assign a bunch of common error codes.

Reply:
It’s just fine.